_posts/2025-03-13-debezium-3-1-beta1-released.adoc

```diff
@@ -0,0 +1,270 @@
+---
+layout: post
+title:  Debezium 3.1.0.Beta1 Released
+date:   2025-03-13
+tags: [ releases, mongodb, mysql, mariadb, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, spanner, jdbc, informix, ibmi ]
+author: ccranfor
+---
+
+We are pleased to announce the first beta release of Debezium 3.1, **3.1.0.Beta1**.
+
+This release includes a myriad of features and improvements, including but not limited to our first official release of Debezium Server UI, CloudEvent traceparent support, new features for Debezium's PubSub and RabbitMQ sinks, schema access in the WASM transformation, and much, much more.
+Let's dive in and take a look at all these new features and improvements.
+
++++<!-- more -->+++
+
+* link:#new-features-and-improvements[New Features and Improvements]
+* link:#other-changes[Other Changes]
+
+[id="new-features-and-improvements"]
+== New features and improvements
+
+The upgrade to Debezium **3.1.0.Beta1** introduces several new features and improvements in several components:
+
+=== First release of Debezium Server UI
```

Comment:
We always referred to it as `Debezium platform` or `Debezium management platform` should we use the same here? Maybe `Debezium Platform (Server UI)`? WDYT?

Reply:
Agreed, and it looks ilke I need to fix the videos from being obnoxiously large.